doc/newsboat.asciidoc

```diff
@@ -264,17 +264,17 @@ are met:
 * none of `-u`, `-c`, `-C` options were specified; and
 
 * the urls file doesn't exist (neither _~/.newsboat/urls_ nor
-  _$XDG_CONFIG_DIR/newsboat/urls_).
+  _$XDG_CONFIG_HOME/newsboat/urls_).
```

Comment:
Good catch.

I've seen before that `$XDG_CONFIG_DIR` wasn't defined on my system but never really thought anything about it.
I guess this explains it.